server/package.json

```diff
@@ -15,6 +15,11 @@
     "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix client && npm run build --prefix client",
     "prettier": "npx prettier --write \"**/*.{js,html}\""
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged"
+    }
+  },
```

Comment:
🤔  I believe you could manage in global scope  otherwise hand you have to duplicate it per app

Reply:
For now, the server directory will be the top directory of the app and we won't have another `package.json` above it. In the future when we migrate to another architecture we will back to this matter.